Quotation.Rmd

```diff
@@ -642,7 +642,7 @@ bquote(-.(xyz) / 2)
   is crucial for functions that evaluate code in the context of a data frame,
   like `subset()` and friends.
 
-Instead functions that quote an argument use some other technique to allow indirect specification. Rather than using use unquoting all base R approaches selectively turn quoting off, so I call them __non-quoting__ techniques.
+Instead functions that quote an argument use some other technique to allow indirect specification. Rather than unquoting, all base R approaches selectively turn quoting off, so I call them __non-quoting__ techniques.
```

Comment:
```suggestion
Instead functions that quote an argument use some other technique to allow indirect specification. Rather than using unquoting, all base R approaches selectively turn quoting off, so I call them __non-quoting__ techniques.
```